### ISSUE TITLE
Can't change language in Qt

### ISSUE THREAD

**Issue by SevenSpheres** [CelestiaProject/Celestia#627]
**Describe the bug**
Remember #441? The method to change Celestia's language described there didn't work for me. I just tested it with Windows native Celestia and it does work there, it's just the Qt version that it doesn't work with.

**To Reproduce**
On Windows with the system language set to English:
1. Run cmd.exe
2. cd to directory with celestia-qt.exe
3. Type `set LC_MESSAGES=be` (or another language code) and press Enter
4. Type `celestia-qt` and press Enter
5. Celestia starts, still in English

**Expected behavior**
Celestia starts in the language specified after `LC_MESSAGES`. This is the behavior in the Windows native version.

**Desktop (please complete the following information):**
 - OS: Windows
 - Frontend: Qt
 - Version: 1.7.0

**Additional context**
It should probably be possible to specify language in celestia.cfg; see #475

**Comment by LukeCEL** [comment]
> It should probably be possible to specify language in celestia.cfg; see #475

I think Celestia is made to automatically set the language to what the computer uses. Specifying language in celestia.cfg would make it hard for Celestia users who are unfamiliar with English, so it shouldn't be the _only_ way to specify the language.

**Comment by eyvallah** [comment]
for qt version linux/macOS, you can start in terminal by `LC_MESSAGES=be ./celestia-qt`, I tried, it worked for me.

I would say it should a low priority feature to change language

**Comment by hv-work** [comment]
`LC_MESSAGES` requires full locale code like `be_BY@UTF-8`, short codes doesn't work always. Another issue is that for `LC_MESSAGES` locale used should be installed on your system.

As we use GNU gettext, `LANGUAGE` environmental variable should be used instead. It accepts short codes and locale data is not required to be installed.

examples:

```
LANGUAGE=uk ./src/celestia/qt/celestia-qt
LANGUAGE=zh_TW ./src/celestia/qt/celestia-qt
```

or
```
export LANGUAGE=zh_TW
./src/celestia/qt/celestia-qt
```

In case of windows use `set`:
```
set LANGUAGE=zh_TW
./src/celestia/qt/celestia-qt
```

**Comment by SevenSpheres** [comment]
Do you understand what I'm saying? If I do something with the Windows native version, it works. If I do the same thing with the Qt version, it doesn't work. This is clearly a bug in the Qt version.

**Comment by LukeCEL** [comment]
@eyvallah for me (using MacOS) this only worked when I used `LANGUAGE` instead of `LC_MESSAGES`. When I used `LC_MESSAGES`, I got this message:

> Warning: Failed to set locale category LC_MESSAGES to be.

Also, how do I change the language for Celestia 1.6.1? Neither `LANGUAGE` or `LC_MESSAGES` work for me.

**Comment by eyvallah** [comment]
@LukeCEL  for 1.6.1, you can use  Celestia.app/Contents/MacOS/Celestia -AppleLanguages '(de)'

**Comment by LukeCEL** [comment]
That works, thank you!
![detranslation](https://user-images.githubusercontent.com/45546434/75612721-4555ea80-5af4-11ea-9a4c-cf5f7fb54f4a.png)

**Comment by 375gnu** [comment]
Wrong path in Qt frontend on windows:
```
c:/Program Files (x86)/celestia/share/locale/be/LC_MESSAGES/celestia_constellations.mo
```

**Comment by SevenSpheres** [comment]
Fixed by #638

**Comment by SevenSpheres** [comment]
Fixed by #638
